app/static/js/changeProposalAction.js

```diff
@@ -0,0 +1,31 @@
+
+// withdrawing a course
+course = null

```

Comment:
Figure it out without using a global variable

Reply:
Done!